src/components/mdCheckbox/mdCheckbox.scss

```diff
@@ -19,6 +19,7 @@ $checkbox-touch-size: 48px;
 
     &:focus {
       outline: none;
+      box-shadow: 0 0 0 2px rgba(#000, .14);
```

Comment:
otherwise there will be no visual indication that the checkbox is focused. keyboard toggling is kinda hard without that ;)